index.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>MY PORTFOLIO</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Inter&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="style.css" />
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.1/css/all.css" integrity="sha384-vp86vTRFVJgpjF9jiIGPEEqYqlDwgyBgEF109VFjmqGmIY/Y4HV4d3Gp2irVfcrp" crossorigin="anonymous">
+  </head>
+  <body>
+    <header class="hero">
+      <nav class="herocontent">
+        <a href="#" class="logo">Welcome</a>
+        <ul class="nav-list">
+          <li><a href="#">Portfolio</a></li>
+          <li><a href="#">About</li>
+          <li><a href="#">Contact</a></li>
+        </ul>
+        <img for="check" class="hambuger-btn"
+           class="fas fa-bars"
+        />
+      </nav>
```

Comment:
- [ ] [**OPTIONAL**] Please make sure the hamburger button is showing up on the mobile version, and the nav bar is not affected by the background image:
- [ ] - [ ] [**OPTIONAL**] As optionally required by the previous code reviewer. This container is set to ```display: flex``` as recommended by the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/1_setup.md#:~:text=In%20order%20to%20lay%20out%20the%20elements%20on%20the%20page%20you%20should%20use%20Flexbox%20in%20all%202%20sections.) factory_worker
:warning: :warning: :warning: 
**Please note that this option is crucial for the project, It's optional for now only because the previous reviewer didn't catch it** . Kindly make sure you change that.

Your design : 

![Screenshot from 2022-12-03 11-25-12](https://user-images.githubusercontent.com/80458199/205433927-90ba36e5-af8f-4665-aa7b-7e258ba0fa37.png)

Figma design: 

![Screenshot from 2022-12-03 11-28-37](https://user-images.githubusercontent.com/80458199/205434061-b47d3e08-a7c1-4c46-83bb-feba62e8a06d.png)